pkg/cmd/cli/cmd/exporter.go

```diff
@@ -148,7 +148,7 @@ func (e *DefaultExporter) Export(obj runtime.Object, exact bool) error {
 		return deployrest.Strategy.Export(ctx, obj, exact)
 
 	case *buildapi.BuildConfig:
-		buildconfigrest.Strategy.PrepareForCreate(ctx, obj)
+		buildconfigrest.GroupStrategy.PrepareForCreate(ctx, obj)
```

Comment:
leave this as legacy.  The new endpoint will have it, the old one won't, and you don't want to twiddle it.